posthog/temporal/workflows/metrics.py

```diff
@@ -0,0 +1,24 @@
+from temporalio import activity
+from temporalio.common import MetricCounter
+
+
+def get_rows_exported_metric() -> MetricCounter:
+    return activity.metric_meter().create_counter("batch_export_rows_exported", "Number of rows exported.")
```

Comment:
I verified that `create_counter` (and others) are reusable, i.e. they point into the metrics registry and if you "create" another metric with the same name/attrs it can be used to incr the existing metric. This is necessary because we're no longer registering metrics in global scope. We use the `activity` to reach the runtime and create a metric as needed to increment (and it comes with free context per activity, which is why we don't have to spell out the `destination` now).

Reply:
Based on this, I'm thinking whether we should write a class to hold the metrics, clickhouse client, and run the activities as methods (Perhaps `BatchExportActivities`). Already wanted to do that just for the clickhouse client, but now we have another piece of state to share across all activities.

However, I'm completely fine with shipping metrics like this, and working on them in this module. Can make a decision on the refactoring later.